### ISSUE TITLE
Default Activity not found

### ISSUE THREAD

**Issue by Sissi91** [martoreto/aauto-sdk-demo#3]
Hei Guys

I tried to import and use the demo app. But I'm not able to load it into the emulator or into my device as it couldn't find a "Default Activity" in the AndroidManifest. Any suggestions what I'm doing wrong? Or what I need to do that I can work on it? Thanks!

**Comment by nerone-github** [comment]
Probably you found out but just adding the info on how to do that. Just edit the launch configuration and select "Nothing" instead of "Default Activity"

**Comment by martoreto** [comment]
Sorry I didn't answer earlier, I didn't know I have to watch my own repositories to get notifications about new issues.

Closing this one.

**Comment by Francis-Smith** [comment]
Hi

Thanks for the great work!

I hit the same problem as Siss91.  I followed nerone's instructions but I'm still unable run run the app.  

I'm trying to deploy this app in the same way that I have successfully deployed regular Media/Messenger apps; I'm deploying to a Samsung S5 running the Android Auto app.  After deploying with Launch Activity set to "Nothing", I don't get anything happening on the phone - no app available and nothing happening in Android Auto.

Any idea what am I missing?

Many thanks
